### PR TITLE
Add Avatto/Tuya 1/2-ch dimmer support

### PR DESCRIPTION
**Pull request by NH-Networks**
#Avatto 1/2-ch dimmer support #3141

TS0601_TZE204_o9gyszw2
TS0601_TZE204_5cuocqty

## Checklist

- [x] The changes are tested and work correctly
- [x] `pre-commit` checks pass / the code has been formatted using Black
- [x] Tests have been added to verify that the new code works

### CODE REVIEW

File: zhaquirks/tuya/ts0601_dimmer.py [hunk 1]
```diff
@@ -153,6 +153,7 @@ class TuyaSingleSwitchDimmerGP(TuyaDimmerSwitch):
             ("_TZE200_y8yjulon", "TS0601"),
             ("_TZE204_n9ctkb6j", "TS0601"),  # BSEED
             ("_TZE204_vevc4c6g", "TS0601"),  # BSEED
+            ("_TZE204_5cuocqty", "TS0601"),  #Avattto ZDMS16-1
```
Comment (TheJulianJES) [inline zhaquirks/tuya/ts0601_dimmer.py]:
```suggestion
            ("_TZE204_5cuocqty", "TS0601"),  # Avattto ZDMS16-1
```

File: zhaquirks/tuya/ts0601_dimmer.py [hunk 2]
```diff
@@ -214,6 +215,7 @@ class TuyaDoubleSwitchDimmerGP(TuyaDimmerSwitch):
             ("_TZE200_fjjbhx9d", "TS0601"),
             ("_TZE200_gwkapsoq", "TS0601"),  # Loratap
             ("_TZE204_zenj4lxv", "TS0601"),
+            ("_TZE204_o9gyszw2", "TS0601"),  #Avattto ZDMS16-2
```
Comment (TheJulianJES) [inline zhaquirks/tuya/ts0601_dimmer.py]:
```suggestion
            ("_TZE204_o9gyszw2", "TS0601"),  # Avattto ZDMS16-2
```

**Review by TheJulianJES — COMMENTED** [review]
Z2M has these listed as a separate device. I'll need to have a closer look at that.
But if these work fine for you, this might be good to go.

**Comment by NH-Networks** [comment]
> Z2M has these listed as a separate device. I'll need to have a closer look at that. But if these work fine for you, this might be good to go.

Working perfectly, it is same device but just 1ch and 2ch.. have them both and work fine this way

**Review by TheJulianJES — APPROVED** [review]
Thanks!

**Pull request merged**